src/app/pages/portal/portal.component.scss

```diff
@@ -79,93 +80,145 @@ mat-icon.disabled {
 igo-map-browser {
   width: 100%;
   height: 100%;
-}
 
-igo-map-browser ::ng-deep .ol-attribution {
-  left: 50px;
-  bottom: $igo-margin;
-  text-align: left;
-  padding: 0;
-  margin-right: 90px;
-  background-color: rgba(255, 255, 255, 0);
-  width: calc(100% - 500px);
-
-  &.ol-logo-only {
-    height: inherit;
-  }
-  &.ol-collapsed {
-    background: none;
-    button {
-      transform: none;
+  &.has-expansion-panel igo-baselayers-switcher {
+    bottom: 47px;
+  }
+
+  &.sidenav-offset-baselayers igo-baselayers-switcher {
+    left: 405px;
+  }
+
+  &.sidenav-offset ::ng-deep .ol-overlaycontainer-stopevent {
+    left: 400px;
+    width: calc(100% - 380px);
+
+    .ol-scale-line {
+      left: 40%;
+    }
+
+    @include mobile {
+      display: none;
     }
   }
-  button {
-    transform: rotate(180deg);
-    background-color: white;
-    cursor: pointer;
+
+  &.expansion-offset ::ng-deep .ol-overlaycontainer-stopevent {
+    bottom: $app-expansion-panel-height;
   }
-  &.ol-uncollapsible {
-    height: auto;
+
+  &.expansion-offset-maximized ::ng-deep .ol-overlaycontainer-stopevent {
+    bottom: $app-expansion-panel-height-maximized;
   }
-  ul {
-    background-color: rgba(255, 255, 255, 0.6);
-    padding: 2px 0.5em;
+
+  &.toast-offset-scale-line
+    ::ng-deep
+    .ol-overlaycontainer-stopevent
+    ::ng-deep
+    .ol-scale-line {
+    bottom: 50px;
   }
-}
 
-igo-map-browser.has-expansion-panel igo-baselayers-switcher {
-  bottom: 47px;
-}
+  &.toast-offset-attribution
+    ::ng-deep
+    .ol-overlaycontainer-stopevent
+    ::ng-deep
+    .ol-attribution {
+    @include mobile {
+      bottom: 50px;
+    }
+  }
 
-igo-map-browser.sidenav-offset-baselayers igo-baselayers-switcher {
-  left: 405px;
-}
+  ::ng-deep {
+    .ol-attribution {
+      left: 50px;
+      bottom: $igo-margin;
+      text-align: left;
+      padding: 0;
+      margin-right: 90px;
+      background-color: rgba(255, 255, 255, 0);
+      width: calc(100% - 500px);
+
+      &.ol-logo-only {
+        height: inherit;
+      }
+      &.ol-collapsed {
+        background: none;
+        button {
+          transform: none;
+        }
+      }
+      button {
+        transform: rotate(180deg);
+        background-color: white;
+        cursor: pointer;
+      }
+      &.ol-uncollapsible {
+        height: auto;
+      }
+      ul {
+        background-color: rgba(255, 255, 255, 0.6);
+        padding: 2px 0.5em;
+      }
+    }
+    .ol-overlaycontainer-stopevent {
+      position: absolute;
+      width: 100%;
+      left: 0;
+      bottom: 0;
+    }
+  }
 
-igo-map-browser ::ng-deep .ol-overlaycontainer-stopevent {
-  position: absolute;
-  width: 100%;
-  left: 0;
-  bottom: 0;
-}
+  .map-actions {
+    position: absolute;
+    display: flex;
 
-igo-map-browser.sidenav-offset ::ng-deep .ol-overlaycontainer-stopevent {
-  left: 400px;
-  width: calc(100% - 380px);
+    .spinner-spacer {
+      height: 40px;
```

Comment:
is it possible to retrieve the spinner size from the lib?

$igo-spinner-size: 40px;

Reply:
Not for now without changes to the lib